src/modules/UI/scenes/SendConfirmation/selectors.js

```diff
@@ -6,10 +6,9 @@ import { type GuiMakeSpendInfo } from '../../../../reducers/scenes/SendConfirmat
 import { type RootState } from '../../../../types/reduxTypes.js'
 import { convertNativeToExchange } from '../../../../util/utils.js'
 import { getExchangeDenomination } from '../../../Settings/selectors.js'
-import { convertCurrency, getSceneState } from '../../selectors.js'
+import { convertCurrency } from '../../selectors.js'
```

Comment:
do we have a task to remove sendComfirmation scene and connector?

Reply:
Oh, dang, you are right! If the scene is going away, maybe I should do that first, and then rebase this on top.

Reply:
Created a separate task to get rid of the send confirmation scene.